admin/services/data-access/azure-blob.data.service.js

```diff
@@ -1,32 +1,37 @@
 'use strict'
 
-const azureStorage = require('azure-storage')
-const bluebird = require('bluebird')
+const { BlobServiceClient } = require('@azure/storage-blob')
 const config = require('../../config')
 
-let blobService
+// TODO cache the client once loaded, or is it better to recreate each time?
```

Comment:
As blob uploads are fairly uncommon let's agree to recreate the service as needed each time?